assets/js/vue/components/search/ResultSet.vue

```diff
@@ -42,13 +42,26 @@
                 </div>
             </div>
         </section>
-        <b-pagination
-                v-model="currentPage"
-                :total-rows="rows"
-                :per-page="formValues.perPage"
-                class="bg justify-content-center"
-                style="margin-bottom: 100px;">
-        </b-pagination>
+
+        <div class="container">
```

Comment:
If you move this div in the `<section>` above, it looks better. There is a stray gray box in the front.

Reply:
Moved it